packages/styles/combobox.css

```
@@ -0,0 +1,246 @@
+:root {
+  --combobox-input-border-color: var(--field-border-color);
+  --combobox-input-background-color: var(--field-background-color);
+  --combobox-input-border-focus-color: var(--field-border-color-focus);
+  --combobox-input-border-hover-color: var(--field-border-color-focus-hover);
+  --combobox-input-border-focus-glow-color: var(
+    --field-border-color-focus-glow
+  );
+  --combobox-input-error-border-color: var(--field-border-color-error);
+  --combobox-input-error-focus-border-color: var(
+    --field-border-color-error-hover
+  );
+  --combobox-input-error-focus-border-glow-color: var(
+    --field-border-color-focus-glow
+  );
+  --combobox-input-font-size: var(--field-font-size);
+  --combobox-input-font-color: var(--field-content-color);
+  --combobox-listbox-border-color: var(--gray-40);
+  --combobox-listbox-background-color: #fff;
```

I think we have the variable of `--white`.
```suggestion
  --combobox-listbox-background-color: var(--white);
```

---

`#fff` is shorter than `--white`. We can't deprecate it because its in use, but there's little to no advantage to defining white as a var because that definition will never change.